lib/accumulate_distribute/util/generate_order.js

```diff
@@ -52,6 +53,14 @@ const generateOrder = (instance = {}) => {
       meta: { _HF: 1 }
     })
   } else if (orderType === 'LIMIT') {
+    console.log('ORDER------------------------->', new Order({
```

Comment:
should we keep this console.log or it was just for dev purposes?

Reply:
for dev purposes, forgot about it.